dev/breeze/tests/test_selective_checks.py

```diff
@@ -973,7 +973,7 @@ def test_expected_output_pull_request_v2_7(
                 "run-tests": "true",
                 "docs-build": "true",
                 "skip-pre-commits": "check-airflow-provider-compatibility,check-extra-packages-references,check-provider-yaml-valid,identity,lint-helm-chart,mypy-airflow,mypy-dev,mypy-docs,mypy-providers,validate-operators-init",
-                "docs-list-as-string": "apache-airflow docker-stack",
```

Comment:
Hmm. Are you sure this is right ?

Reply:
I am not happy about this but if the build is running with `"apache-airflow docker-stack"` then it seems the docsbuild catches Sphinx inventories from "current" providers and then some link breaking changes generate fails. See for example here: https://github.com/apache/airflow/actions/runs/13887634551/job/38854668394?pr=47755

Therefore to fix docs build my proposal in this PR is to build _all_ documentation even though only core is published/needed.

But maybe I am not creative/experienced enough to understand what is broken in the docsbuild.